ico/cmd/distributetokens.py

```diff
@@ -170,7 +171,7 @@ def main(chain, address, token, csv_file, limit, start_from, issuer_address, add
 
             transaction = {
                 "from": address,
-                "gasPrice": int(web3.eth.gasPrice * 1.5)
+                "gasPrice": gas_price * 1000000000
```

Comment:
yep - it all seems good. - ztane have placed some comments already.

But these numbers with a lot of zeros are really hard to read correctly, and they are very critical - I'd suggest creating a global constant of one million - ie - a line in the beggining of the file like `MILLION = int(10e6)` 

and them write these gas_price expressions as: `gasprice * 100 * MILLION`
( `int(10e8)` would work, of course, but I find 100 * MILLION more readable)

Reply:
I have totally different view on this, even though it's a rather small matter. I would say having constant like `MILLION = int(10e6)` is not only not correct (`10e6` is actually `10**7` => 10 millions) but also doesn't really improve the readability of the code.

The context of this part of code is to convert `gas_price` from wei to gigawei. So the thing that developers really care about is how many `0` there are, which is the reason why we can say `1000000000` is a hard number to read => easy to make mistake. And for that reason alone it's best to use exponentiation.

And if you want to take it a little bit further, then you can define an utility function like:
```
def wei_to_gwei(price):
    return price * 10 ** 9  # or price * (10 ** 9) if you don't trust your memory on operator precedence 
```
then it's readable, understandable, reusable and testable.

Reply:
I would not say it is small matter - and neither that it is "totally different" view. The major  problem, of course is reading the zeros.  I agree that putting the exponentiation directly will make the number of zeros more apparent (and less prone to subtle catastrophic mistakes as the one I just made, of writting 10e6 thinking about  10 ** 6)

Reply:
There is the other matter that `int(1e6)` is *not* a compile-time constant, unlike `10 ** 6`.